src/transformers/__init__.py

```diff
@@ -849,6 +851,17 @@
 
     # PyTorch models structure
 
+    _import_structure["models.fan"].extend(
+        [
+            "FAN_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "FANForImageClassification",
+            "FANForSemanticSegmentation",
+            "FANLayer",
+            "FANModel",
+            "FANPreTrainedModel",
+            "load_tf_weights_in_fan",
+        ]
+    )
```

Comment:
```suggestion
    _import_structure["models.fan"].extend(
        [
            "FAN_PRETRAINED_MODEL_ARCHIVE_LIST",
            "FANForImageClassification",
            "FANForSemanticSegmentation",
            "FANModel",
            "FANPreTrainedModel",
        ]
    )
```
You can remove `FANLayer` and `load_tf_weights_in_fan` everywhere from the init.